plugins/node/opentelemetry-instrumentation-mysql/test/index.test.ts

```diff
@@ -152,6 +152,66 @@ describe('mysql@2.x', () => {
         });
       });
     });
+
+    it('db.statement should have ? instead of quoted words when using default serializer ', done => {
```

Comment:
This test checks only the query `object` invocation. Below you added tests for both `object` and `string`. Should we also add a test for the default config with string parameter for symmetry?